frontend-full-build/package-lock.json

```diff
@@ -20597,6 +20597,12 @@
         "decamelize": "^1.2.0"
       }
     },
+    "node_modules/jquery": {
```

Comment:
please double check package-lock.json file.  Did we change the package.json

Reply:
resolved- my package.lock should match main now.